Automattic/wp-calypso
# Docs: Move CONTRIBUTING.md to docs/

#### Changes proposed in this Pull Request

* Docs: Move `CONTRIBUTING.md` from `.github/` to `docs/`

The motivation for this is #35833 -- to enable webpack loaders to find this file (in a non-hidden directory).

GitHub should continue to be able to find this file:

> To help your project contributors do good work, you can add a file with contribution guidelines to your project repository's root, `docs`, or `.github` folder. When someone opens a pull request or creates an issue, they will see a link to that file.

[(Source)](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors)

#### Testing instructions

- Go to `calypso.localhost:3000/devdocs`, and click on "Contributing" in the sidebar. Verify that the link still works.
- Grep the codebase to verify there are no other references to `.github/CONTRIBUTING.md`.
